EWD998.tla

```diff
@@ -125,40 +128,14 @@ Next ==
         \/ InitiateToken(i)
         \/ PassToken(i)
 
-Invariant ==
-    \A n \in Node: network[n] = 0
 
-Property2 ==
-    [Next]_vars
-
-    \*  ( [A]_v )  <=>  (A \/ UNCHANGED v)
 -------------------
 
-terminationDetected ==
-    /\ token.pos = 0
-    /\ token.tainted = FALSE
-    /\ token.value = 0
-    /\ ~active[0]
-    /\ ~msgSentNotTainted[0]
-
 Spec ==
     /\ Init
     /\ [][Next]_vars
-    \* /\ <>terminationDetected
-    \* /\ WF_vars(Next)
-    /\ \A i \in Node: WF_vars(PassToken(i) \/ InitiateToken(i))
-    
+    /\ (\A i \in Node: WF_vars(PassToken(i) \/ InitiateToken(i) \/ Terminate(i))) 
```

Comment:
I'm also wondering if we should enforce `Terminate` here, as our termination algorithm should ideally be safe and live even for a system that never terminates? But realistically a valid computation task should terminate.

Reply:
I'm also thinking if WF termination might be too strong as it might ask the state chain to terminate sooner?